docs/guides/react-native.md

```diff
@@ -9,32 +9,188 @@ a host device.
 This can be difficult when you want your test suite to run with typical Continuous Integration servers
 such as Travis.
 
-A pure JS mock of React Native exists and can solve this problem in the majority of use cases.
+To use enzyme to test React Native, you currently need to configure an adapter, and load an emulated DOM.
 
-To install it, run:
+## Configuring an Adapter
 
-```bash
-npm i --save-dev react-native-mock
+While a React Native adapter is [in discussion](https://github.com/airbnb/enzyme/issues/1436),
+a standard adapter may be used, such as "enzyme-adapter-react-16":
+
+```
+import Adapter from "enzyme-adapter-react-16";
+
+Enzyme.configure({ adapter: new Adapter() });
+```
+
+## Loading an emulated DOM with JSDOM
+
+To use enzyme's `deep` until a React Native adapter exists, an emulated DOM must be loaded.
+
+While some have had success with [react-native-mock-renderer](https://github.com/Root-App/react-native-mock-render),
+the recommended approach is to use [https://github.com/tmpvar/jsdom](JSDOM),
+as documented for enzyme at the [JSDOM](https://airbnb.io/enzyme/docs/guides/jsdom.html) documentation page.
+
+JSDOM will allow all of the `enzyme` behavior you would expect, and it will result in reasonable snapshots if
```

Comment:
let's note here that enzyme has no snapshot support besides `.debug()`

Reply:
Do you mean if jsdom isn't used? In our React Native boilerplate where we are using jsdom, we get better output from just expecting on the wrapper object itself:

https://github.com/ProminentEdge/mobile-boilerplate/blob/4994f554e64c63c7b10252baeb62c31a7cc6c5c1/src/screens/todo-list/todo-list.test.tsx#L141

If expecting with `expect(wrapper.debug())` we get e.g.

```
      <Container virtual={[undefined]} style={{...}}>
```

Where as if you expect with `expect(wrapper)` it will show us more information, like what styles were applied

```
      <Container
        style={
          Object {
            "backgroundColor": "#fff",
            "flex": 1,
            "height": 1334,
          }
        }
      >
```

Reply:
Do you mean that it isn't supported without `jsdom`? Because we get better output with `jsdom` if we just expect on the wrapper.

In our React Native boilerplate where we are using jsdom, we get better output from just expecting on the wrapper object itself:

https://github.com/ProminentEdge/mobile-boilerplate/blob/4994f554e64c63c7b10252baeb62c31a7cc6c5c1/src/screens/todo-list/todo-list.test.tsx#L141

If expecting with `expect(wrapper.debug())` we get e.g.

```
      <Container virtual={[undefined]} style={{...}}>
```

Where as if you expect with `expect(wrapper)` it will show us more information, like what styles were applied

```
      <Container
        style={
          Object {
            "backgroundColor": "#fff",
            "flex": 1,
            "height": 1334,
          }
        }
      >
```

Reply:
That makes sense, I updated the wording, let me know how that sounds: 

```
JSDOM will allow all of the `enzyme` behavior you would expect. While Jest snapshot testing can be used with
this approach as well, it isn't encouraged and is only supported through `wrapper.debug()`.
```

Reply:
Sounds great, thanks.